core/pom.xml

```diff
@@ -455,11 +461,6 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
-		<dependency>
-			<groupId>cglib</groupId>
-			<artifactId>cglib-nodep</artifactId>
-			<version>3.3.0</version>
-		</dependency>
```

Comment:
CGLIB is unmaintained, does not work with JDK17, and we do not appear to need it anymore for anything we use.